### ISSUE TITLE
ADD new move

### ISSUE THREAD

**Issue by NEX-S** [m4xshen/autoclose.nvim#3]
For example:
```c
#include "stdio.h|"               press  "                    #include "stdio.h"|
#include <stdio.h|>             press  >                   #include <stdio.h>|
etc...
```

**Comment by m4xshen** [comment]
This feature was added in the latest commit 3d3ddb8cdf100f1ccc7a77c20bd4a8223a31dfa5. Thank you for the feedback.

**Comment by m4xshen** [comment]
Fix these bugs in c283288e75b83f11f9cbc227e997e02cc24ffc9c. Feel free to give me more feedback.

**Comment by m4xshen** [comment]
Oh... I see. Could you please open a new issue? This is different to the previous issues.
